src/components/pages/request-form/request-form.component.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import { KeyboardDatePicker } from '@material-ui/pickers';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Chip from '@material-ui/core/Chip';
+import Input from '@material-ui/core/Input';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+import useStyles from './request-form.styles';
+
+const RequestForm = () => {
+  const classes = useStyles();
+  const [values, setValues] = React.useState({
+    age: '',
```

Comment:
What are these keys? Where come from?

Reply:
It is for the "Holiday Type" select. I'll change the name, it was in the material-ui example.
`<InputLabel>Holiday Type</InputLabel>
            <Select
              **value={values.age}**
              required
              onChange={handleChangeB}
              inputProps={{
                name: 'age',
                id: 'age-simple',
              }}
            >`

Reply:
That's right, but in the example the user can select **age** from the dropdown menu. In this case you can select a **holiday type** from the dropdown not an **age**.
The other problem is the initial state in the useState, you should not use object, because you'll need a string.

Reply:
This will be a valid option for this case:
`const [holidayType, setHolidayType] = React.useState('');`